01-messaging.md

```diff
@@ -325,11 +325,11 @@ The 2-byte `len` field indicates the number of bytes in the immediately followin
 The channel is referred to by `channel_id`, unless `channel_id` is 0 (i.e. all bytes are 0), in which case it refers to all channels.
 
 The funding node:
-  - for all error messages sent before (and including) the `funding_created` message:
```

Comment:
nack

I don't think this introduces some improvement on spec, and as usual, the free text gives us the power like humans to interpret the free text in a different way. IMO, are two ways to say the same things, and this change can cause another change in the future for other people who preferer the first style.